api_tests/lib/util.ts

```diff
@@ -15,6 +15,16 @@ export async function sleep(time: number) {
     });
 }
 
+export function seconds_until(time: number): number {
+    const diff = time - Math.floor(Date.now() / 1000);
+
+    if (diff > 0) {
```

Comment:
🙃 *not tested but should work.
```suggestion
    return diff > 0? diff : 0;
```

Reply:
I just moved the code :upside_down_face: